doc/release/upcoming_changes/14771.improvement.rst

```diff
@@ -0,0 +1,6 @@
+``std=c99`` added if compiler is named ``gcc``
+----------------------------------------------
+
```

Comment:
I don't think the blank line is needed here. I edit them out of the final notes.